theory-test-start-admission/lambda.js

```diff
@@ -1,5 +1,34 @@
+require('dotenv').config();
```

Comment:
**STOP ADDING THIS TO PRODUCTION CODE**

'dotenv' is **not** something you need in production

Reply:
done